src/main/java/app/sami/languageWeb/contractRequest/ContractRequestController.java

```diff
@@ -28,4 +35,25 @@ public ContractRequestDto renderDecision(@PathVariable Long contractRequestId,
         return ContractRequestMapper.toContractRequestDto(contractRequestService
                 .renderContractRequestDecision(status.getStatus(), contractRequestId, subject));
     }
+
+    @GetMapping("/contract-requests")
+    public ContractRequestsDto getContractRequests(@RequestParam Long requestId,
```

Comment:
REST calls are always plural, you should not have `contract-request` and `contract-requestS`